apps/pie-docs/src/components/assistive-text/overview/overview.md

```diff
@@ -0,0 +1,214 @@
+    order: 1
+shouldShowContents: true
+---
+
+## Overview
+
+Assistive text helps users understand what kind of information they should enter, how to format it, or any specific requirements related to the input. It can provide examples, suggestions, error messages, validation rules, or any other relevant details that aid in accurate and meaningful input.
+
+The purpose of assistive text is to enhance the user experience by reducing confusion, improving input accuracy, and ensuring users provide the expected information in the desired format.
+
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/overview.svg",
+    alt: "Assistive text that is highlighting an error underneath a text input."
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+          "Always use in combination with a form element to provide additional information or feedback."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "If displaying success or error variants, don’t show until the user has finished entering information."
+        ]
+    }
+} %}
+
+---
+
+## Anatomy
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/anatomy.svg",
+    alt: "Anatomy of an assistive text.",
+    width: 195
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Icon:** Visually supports the type of message. Only available for error and success validation.",
+        "**String:** Provides informative information to the user."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+Shown when instructional information is required to help the user complete a form element.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/variations-default.svg",
+    alt: "Assistive text component in default state.",
+    width: "85px"
+} %}
+
+### Error
+
+Shown when an input fails validation, and feedback is required on an individual form.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/variations-error.svg",
+    alt: "Assistive text component in error state.",
+    width: "110px"
+} %}
+
+### Success
+
+Shown when an input passes validation, and feedback is required on an individual form.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/assistive-text/variations-success.svg",
+    alt: "Assistive text component in success state.",
+    width: "110px"
+} %}
+
+---
+
+## Content
+
+All content should use sentence case.
+
+### Default
+
+- Should provide additional instructional information that helps the user fill out a form field, as clearly and succinctly as possible.
+- Use sentence case.
+
+### Error
+
+- Should describe the error and inform the user on how to fix it, as clearly and succinctly as possible.
+- Use sentence case.
+
+### Success
+
+- Should describe the success as clearly and succinctly as possible.
+- Use sentence case.
```

Comment:
I think the repetition is a little over the top, perhaps 😄

Reply:
I discussed it with Rachel and it's currently how we do thing. Maybe some changes will be added in future, but for now all good